doc/rtd/introduction.rst

```diff
@@ -0,0 +1,116 @@
+Introduction to cloud-init
+**************************
+
+Managing and configuring cloud instances and servers can be a complex
+and time-consuming task. Cloud-init is an open source initialisation tool that
+was designed to make it easier to get your systems up and running with a
+minimum of effort, and already configured according to your needs.
```

Comment:
Nit: I think grammar rules say don't put a comma before the `and` if what follows is a dependent clause

Reply:
Fair - I'm keeping the comma but removed the "and"